packages/dist-util/index.js

```diff
@@ -6,15 +6,28 @@
 'use strict';
 
 const path = require('path');
+const util = require('util');
+const semver = require('semver');
+
+/**
+ * Make sure node version meets the requirement. This file intentionally
+ * only uses ES5 features so that it can be run with lower versions of Node
+ * to report the version requirement.
+ */
+function checkNodeVersion(range) {
```

Comment:
AFAICT, `range` is not used anywhere. Should we remove it and simplify the implementation a tiny bit?

Reply:
It's used here - https://github.com/strongloop/loopback-next/blob/ece15edca2fb3fd5c177967b737bdc9e54365647/packages/cli/bin/cli-main.js#L12-L13